jerry-core/include/jerryscript-port.h

```diff
@@ -62,7 +77,7 @@ typedef enum
  *
  * Example: a libc-based port may implement this with exit() or abort(), or both.
  */
-void jerry_port_fatal (jerry_fatal_code_t code) __attribute__((noreturn));
+__noreturn void jerry_port_fatal (jerry_fatal_code_t code);
```

Comment:
Is there a specific reason to move the `__noreturn` attribute to the beginning of the declaration from the end?

Reply:
What syntax is accepted? Is `__declspec() type funcname()` the only correct way? In jrt.h, several functions are declared as `type __noreturn funcname()` suggesting that MSVC also accepts `type __declspec() funcname()`. If that's the case, I suggest to place attributes consistently here, too. (And perhaps even some lines below, for `jerry_port_log`.)